### PR TITLE
Commerce 2359 Inventory entities support fields and sorts

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
Hi @ethanbustad , these changes don't require the baseLine too..

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

File: headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/odata/entity/v1_0/WarehouseEntityModel.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import com.liferay.portal.odata.entity.EntityModel;
+import com.liferay.portal.odata.entity.StringEntityField;
+
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Alessio Antonio Rendina
```
Comment (ethanbustad) [inline headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/odata/entity/v1_0/WarehouseEntityModel.java:27]: is this the right author? :)

**Review by ethanbustad — APPROVED** [review]

File: headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/odata/entity/v1_0/WarehouseEntityModel.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ */
+public class WarehouseEntityModel implements EntityModel {
+
+	public WarehouseEntityModel() {
+		_entityFieldsMap = Stream.of(
+			new StringEntityField("city", locale -> "city"),
```
Comment (marco-leo) [inline headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/odata/entity/v1_0/WarehouseEntityModel.java:33]: @andrea-ale-sbarra please add more fields, like country code and region code, active filed and geolocations

**Pull request merged**